cmd/kube-proxy/app/server.go

```diff
@@ -459,6 +468,12 @@ func serveMetrics(ctx context.Context, bindAddress string, proxyMode kubeproxyco
 
 	configz.InstallHandler(proxyMux)
 
+	if utilfeature.DefaultFeatureGate.Enabled(zpagesfeatures.ComponentFlagz) {
+		if flagzReader != nil {
+			flagz.Install(proxyMux, "kube-proxy", flagzReader)
```

Comment:
likewise, you don't need to check the feature gate here (though in this case you do still need the `nil` check)

Reply:
Updated.